release/Dockerfile.in

```
@@ -27,6 +27,7 @@ CMD { tar xz && \
       ./configure --with-vendored-deps --with-mccs && \
       echo "(${LINKING})" > src/client/linking.sexp && \
       make opam && \
+      chmod u+w _build/default/src/client/opamMain.exe && \
       strip opam ; \
     } >&2 && \
     cat opam
```

Wouldn't it be slightly better to copy it here? (or even potentially move this into `Makefile` - `make opam-stripped` or some such)? Along the lines of:
```suggestion
```

```suggestion
      cp opam opam-stripped && \
      chmod 775 opam-stripped && \
      strip opam-stripped ; \
    } >&2 && \
    cat opam-stripped
```